SUMMARY.md

```diff
@@ -1,34 +1,42 @@
 # Table of contents
 
-* [Web3URL](README.md)
+* [Introduction](README.md)
 
-## Overview
+## Vision
 
-* [Background](overview/background.md)
-* [The Problem](overview/the-problem.md)
-* [Our Solution](overview/our-solution.md)
-* [Examples](overview/examples.md)
+* [Vision](vision/vision.md)
+* [Background](vision/background.md)
+* [The Problem](vision/the-problem.md)
+* [Our Solution](vision/our-solution.md)
+* [Examples](vision/examples.md)
 
-## Basics
+## web3:// URL structure
+* [Basic structure](structure/base.md)
+* [Domain name resolution](structure/domain-name.md)
+* [Resolve mode](structure/resolve-mode.md)
+  * [Auto mode](structure/mode-auto.md)
+  * [Manual mode](structure/mode-manual.md)
+  * [Resource request mode](structure/mode-resource-request.md)
 
-* [Web3URL Schema](basics/web3url-schema.md)
-* [Web3URL Gateways](basics/web3url-gateways.md)
-* [FlatDirectory](basics/flatdirectory.md)
+## web3:// clients
```

Comment:
Should we include the Chrome extension from comfygummy?

Reply:
Yes! I was waiting for an publication on the Chrome store, but yes we can list it right away as it is working by manually installing it. Will update.

Reply:
Ok I have added, and copied the disclaimer from his README into the page. I have contacted ComfyGummy to request if he wants any changes.